app/src/main/java/de/danoeh/antennapod/fragment/AllEpisodesFragment.java

```diff
@@ -320,80 +314,11 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
 
-    private final FeedItemMenuHandler.MenuInterface contextMenuInterface = new FeedItemMenuHandler.MenuInterface() {
-        @Override
-        public void setItemVisibility(int id, boolean visible) {
-            if(contextMenu == null) {
-                return;
-            }
-            MenuItem item = contextMenu.findItem(id);
-            if (item != null) {
-                item.setVisible(visible);
-            }
-        }
-    };
-
-    @Override
-    public void onCreateContextMenu(ContextMenu menu, View v, ContextMenu.ContextMenuInfo menuInfo) {
-        super.onCreateContextMenu(menu, v, menuInfo);
-        AdapterView.AdapterContextMenuInfo adapterInfo = (AdapterView.AdapterContextMenuInfo) menuInfo;
-        FeedItem item = itemAccess.getItem(adapterInfo.position);
-        MenuInflater inflater = getActivity().getMenuInflater();
-        inflater.inflate(R.menu.allepisodes_context, menu);
-
-        if (item != null) {
-            menu.setHeaderTitle(item.getTitle());
-        }
-
-        contextMenu = menu;
-        lastMenuInfo = (AdapterView.AdapterContextMenuInfo) menuInfo;
-        FeedItemMenuHandler.onPrepareMenu(getActivity(), contextMenuInterface, item, true, queuedItemsIds);
-    }
-
-    @Override
-    public boolean onContextItemSelected(MenuItem item) {
-        if (!getUserVisibleHint()) {
-            // we're not visible, don't do anything.
-            return false;
-        }
-        AdapterView.AdapterContextMenuInfo menuInfo = (AdapterView.AdapterContextMenuInfo) item.getMenuInfo();
-        if (menuInfo == null) {
-            menuInfo = lastMenuInfo;
-        }
-        if (menuInfo == null) {
-            Log.e(TAG, "menuInfo is null, not doing anything");
-            return false;
-        }
-
-        FeedItem selectedItem = null;
-
-        // make sure the item still makes sense
-        if (menuInfo.position >= 0 && menuInfo.position < itemAccess.getCount()) {
-            selectedItem = itemAccess.getItem(menuInfo.position);
-        } else {
-            Log.d(TAG, "Selected item at position " + menuInfo.position + " does not exist, only " + itemAccess.getCount() + " items available");
-        }
-
-        if (selectedItem == null) {
-            Log.i(TAG, "Selected item at position " + menuInfo.position + " was null, ignoring selection");
-            return super.onContextItemSelected(item);
-        }
-
-        try {
-            return FeedItemMenuHandler.onMenuItemClicked(getActivity(), item.getItemId(), selectedItem);
-        } catch (DownloadRequestException e) {
-            e.printStackTrace();
-            Toast.makeText(getActivity(), e.getMessage(), Toast.LENGTH_LONG).show();
-            return true;
-        }
-    }
-
     private void onFragmentLoaded() {
         if (listAdapter == null) {
-            listAdapter = new AllEpisodesListAdapter(activity.get(), itemAccess,
+            listAdapter = new AllEpisodesRecycleAdapter(activity.get(), activity.get(), itemAccess,
```

Comment:
Do we really want to change it now though?  (assuming we're going to try to get this in 1.4)